pkgs/swiftgen/swift2objc/Sources/swift2objc/AST/StructSyntax/StructMemberSyntax.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  File.swift
+//  
+//
+//  Created by Mohammad Eid on 13/05/2024.
+//
+
+import Foundation
+
+protocol StructMemberSyntax {
+    var accessModifier: AccessModifier { get }
```

Comment:
I guess it will depend on what approach we'll take when transforming the AST. If it'n gonna be an in-place transformation then yeah a setter is needed. So far I made all fields as constants (using `let`), should I change that to `var`?

Reply:
It's a bit hard to know at this point, but I think we're gonna want them to all be mutable. Otherwise, every time you modify one of the higher level AST nodes, you have to copy all its children.